cmd/statusd/main.go

```diff
@@ -158,6 +168,28 @@ func startCollectingStats(interruptCh <-chan struct{}, nodeManager common.NodeMa
 	}
 }
 
+func syncAndStopNode(nodeManager common.NodeManager) (err error) {
+	log.Println("Node will synchronize and exit")
+	v := *syncAndExit
```

Comment:
Why `v`? :P `value` or `timeout` should be easier to reason about.

Reply:
I'd even move it as a param of the function to make it stateless.

Reply:
`v` is just short for `value`. I can make the change if you like but I don't think it would make a big difference. :)

Reply:
(Basically, I just thought there's no need for a meaningful name for it in that scope.)

Reply:
I am just not a fan of one character variable names except for method receiver names. I don't see any value in shortening them but I see disadvantages :D

Reply:
Alright, will change this as well :)